src/hooks/useHtmlPaste/index.ts

```diff
@@ -1,6 +1,7 @@
 import {useCallback, useEffect} from 'react';
 import Parser from '@libs/Parser';
 import CONST from '@src/CONST';
+import isMobile from '@src/utils/isMobile';
```

Comment:
```suggestion
import isMobile from '@libs/Browser';
```

We can use `isMobile` from @libs/browser